src/Microsoft.Dnx.Tooling/Publish/PublishProject.cs

```diff
@@ -564,9 +569,7 @@ private bool GenerateWebConfigFileForWwwRootOut(PublishRoot root, Runtime.Projec
             var attributesToOverwrite = new[]
             {
                 "processPath",
-                "arguments",
-                "stdoutLogEnabled",
-                "stdoutLogFile"
```

Comment:
Can the httpPlatform element be copied from web.config present in the source wwwroot folder?